common/src/main/java/com/revenuecat/purchases/common/logWrapper.kt

```diff
@@ -0,0 +1,60 @@
+package com.revenuecat.purchases.common
+
+import com.revenuecat.purchases.strings.Emojis
+
+fun log(intent: LogIntent, message: String) {
+    val emojifiedMessage = "${intent.emojiList.joinToString("")} $message"
+    when (intent) {
```

Comment:
Ended up removing `LogLevel` entirely since it's not used with this implementation.